src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/tests/ViewRenderer.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import ReactTestRenderer from 'react-test-renderer';
+import ViewRenderer from '../ViewRenderer';
+
+test('Render view returned from ViewRegistry', () => {
+    const ViewRegistry = jest.mock('../../../services/ViewRegistry', () => {
+        return {
+            getView: jest.fn(() => (<h1>Test</h1>)),
+        };
+    });
```

Comment:
Oh yeah, that's because you are using imports and `ViewRenderer` requires `ViewRegistry` before this mock is being set. You can put the `jest.mock` on the top level of this file and it should work.